### ISSUE TITLE
List outputs are not raised as errors

### ISSUE THREAD

**Issue by viraptor** [awslabs/cfn-python-lint#485]
*cfn-lint version: (`cfn-lint --version`)* 0.9.1

*Description of issue.*

Cloudformation fails with:
```
Aws::CloudFormation::Errors::ValidationError Template format error: The Value field of every Outputs member must evaluate to a String and not a List.
```
but cfn-lint doesn't detect this issue.

Please provide as much information as possible:
```
  "Outputs": {
    "RouteTablesPrivate": {
      "Value": [
        {
          "Ref": "RouteTablePrivate1"
        },
        {
          "Ref": "RouteTablePrivate2"
        }
      ]
    }
  }
```

This should raise a similar issue when linting.

**Comment by kddejong** [comment]
agreed.  We should be checking this.

**Comment by kddejong** [comment]
Is included in the newest release.
